src/com/sun/ts/tests/common/vehicle/ejblitejsf/faces-config.xml

```diff
@@ -21,39 +21,5 @@
               xmlns="https://jakarta.ee/xml/ns/jakartaee" 
               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
               xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-facesconfig_3_0.xsd">
-    <managed-bean>
```

Comment:
I see that we are removing the (ignored) JSF managed bean settings which sounds correct since they are removed for EE 10.  

Since there were no changes to the Managed Beans SPEC for EE 10 (as per https://jakarta.ee/specifications/managedbeans), does that mean that managed beans should continue to work as they did for EE 9?

Reply:
> does that mean that managed beans should continue to work as they did for EE 9?

Yes, JSF Managed Beans are different from Jakarta Annotation Managed Beans

> Or are these tests not using https://jakarta.ee/specifications/annotations/2.1/apidocs/jakarta.annotation/jakarta/annotation/managedbean?

Probably not. Jakarta Annotation Managed Beans are more a concept, and in practice not explicitly used.

Reply:
For reference, we do have 27 references to jakarta.annotation.ManagedBean, some of which are in ejb30/lite test folder:

> cd jakartaee-tck/src
> grep -ri jakarta.annotation.ManagedBean
> com/sun/ts/tests/appclient/deploy/metadatacomplete/testapp/TestBean.java
> com/sun/ts/tests/appclient/deploy/metadatacomplete/testapp/files/TestBean.java.src
> com/sun/ts/tests/jaxrs/platform/managedbean/TSAppConfig.java
> com/sun/ts/tests/jaxrs/platform/managedbean/ManagedBeanRootResource.java
> com/sun/ts/tests/jaxrs/platform/managedbean/StringBuilderProvider.java
> com/sun/ts/tests/signaturetest/signature-repository/jakarta.annotation.sig
> com/sun/ts/tests/signaturetest/caj/CAJSigTest.java
> com/sun/ts/tests/jms/ee20/cditests/ejbweb/MyManagedBean.java
> com/sun/ts/tests/ejb30/misc/moduleName/appclientejb/ModuleMBean.java
> com/sun/ts/tests/ejb30/misc/moduleName/twowars/ModuleMBean.java
> com/sun/ts/tests/ejb30/misc/moduleName/twojars/ModuleMBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/invocationcontext/InvocationContextInterceptorBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/invocationcontext/InvocationContextBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/business/InterceptorOverride34Bean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/business/InterceptorOverrideBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/business/InterceptorBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/lifecyclecdi/OverrideWithPostConstructBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/lifecyclecdi/OverrideBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/lifecycle/InterceptorOverride34Bean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/lifecycle/AManagedBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/lifecycle/InterceptorOverrideBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/mbean/interceptor/lifecycle/InterceptorBean.java
> com/sun/ts/tests/ejb30/lite/packaging/war/datasource/singleton/DataSourceMBean.java.template
> com/sun/ts/tests/ejb30/lite/packaging/war/datasource/singleton/DataSourceRepeatableBean.java.template
> com/sun/ts/tests/ejb30/assembly/appres/common/AppResManagedBean.java
> com/sun/ts/tests/ejb30/assembly/mbean/appclient/TwoManagedBean.java
> com/sun/ts/tests/ejb30/assembly/mbean/appclient/OneManagedBean.java